File: LMG.Fab.Web/ClientApp/app/models/tiers.ts
import { ParamDetail } from "./paramTable";

export class Tiers {
    public PkTiers: number;
    public FkParamDetailType: number;
    public Nom: string;
    public Adresse1: string;
    public Adresse2: string;
    public Adresse3: string;
    public CodePostal: string;
    public Ville: string;
    public Pays: string;
    public NomContact: string;
    public Telephone: string;
    public CodeFournisseurErp: string;
    public CodeEntrepotErp: string;
    public SignatureFicheTechnique: string;
    public TexteCommande: string;
    public Actif: boolean;
    public AdresseEmail: string;
    public OkImprimvert: boolean;
    public PkPrestation: number;
    public TauxHoraire: number;
    public Prestation: Prestation[];
}

export class Prestation {
    public PkPrestation: number;
    public FkTiers: number;
    public FkParamDetailTrfo: number;
    public TauxHoraire: number;
    public Machine: Machine[];
}

export class Machine {
    public PkMachine: number;
    public Nom: string;
    public Tournees: string;
    public Roto: boolean;
    public Coming: boolean;
    public Cameron: boolean;
    public FausseCoupe: number;
    public FkPrestation: number;
    public SousMultiplesCahiers: string;
    public OkNumerique: boolean;
    public MachineGamme: MachineGamme[];
}

export class MachineGamme {
    public PkMachineGamme: number;
    public FkMachine: number;
    public FkParamDetailGamm: number;
    public CodeDatexp: string;
    public Actif: boolean;
}

File: LMG.Fab.Data/Entities/LmgFab/Tiers.cs
using System;
using System.Collections.Generic;

namespace LMG.Fab.Data.Entities.LmgFab
{
    public partial class Tiers
    {
        public Tiers()
        {
            Prestation = new HashSet<Prestation>();
        }

        public int PkTiers { get; set; }
        public int? FkParamDetailType { get; set; }
        public string Nom { get; set; }
        public string Adresse1 { get; set; }
        public string Adresse2 { get; set; }
        public string Adresse3 { get; set; }
        public string CodePostal { get; set; }
        public string Ville { get; set; }
        public string Pays { get; set; }
        public string NomContact { get; set; }
        public string Telephone { get; set; }
        public string CodeFournisseurErp { get; set; }
        public string CodeEntrepotErp { get; set; }
        public string SignatureFicheTechnique { get; set; }
        public string TexteCommande { get; set; }
        public bool? Actif { get; set; }
        public string AdresseEmail { get; set; }
        public bool? OkImprimvert { get; set; }

        public ParamDetail FkParamDetailTypeNavigation { get; set; }
        public ICollection<Prestation> Prestation { get; set; }
    }
}
